src/masonite/validation/__init__.py

```diff
@@ -28,6 +28,7 @@
     none,
     numeric,
     phone,
+    regex,
```

Comment:
Typically when you make an import its better for it to be sorted in alphabetical. It's easier to read. You can't also sort alphabetically because of import hierarchy but when possible I like to keep things clean

Reply:
Yes I agree, will pay attention in next PRs 👍